tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1656,7 +1656,7 @@ feature! {
     }
 
 
-    impl<C, S> Subscribe<C> for Vec<S>
+    impl<C, S> Subscribe<C> for alloc::vec::Vec<S>
```

Comment:
This change is unrelated, but included here because I noticed that
```sh
$ cargo build --no-default-features --features alloc
```
was failing without it.